Jenkinsfile

```diff
@@ -53,6 +53,12 @@ def job = {
               sh "./gradlew --init-script ${GRADLE_NEXUS_SETTINGS} --no-daemon uploadArchivesAll"
           }
       }
+    } else if (config.publish && config.isPreviewJob) {
```

Comment:
i think this can be done with out duplicating the gradle command and the stage definition.